From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@google.com>
Date: Mon, 23 Sep 2019 14:43:39 +0200
Subject: ANDROID: update arm64 gki_defconfig

Update the arm64 gki_defconfig for changes made during the merge of
upstream

Signed-off-by: Greg Kroah-Hartman <gregkh@google.com>
Change-Id: I50d96aa097866a43f305c59954f1242e02d0dac6
---
 arch/arm64/configs/gki_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index ab1dd6d01ec8..01be7843c646 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -384,7 +384,6 @@ CONFIG_FUSE_FS=y
 CONFIG_OVERLAY_FS=y
 CONFIG_MSDOS_FS=y
 CONFIG_VFAT_FS=y
-CONFIG_TMPFS_POSIX_ACL=y
 # CONFIG_EFIVAR_FS is not set
 CONFIG_SDCARD_FS=y
 CONFIG_PSTORE=y
